.circleci/config.yml

```diff
@@ -111,6 +111,41 @@ workflows:
             branches:
               only: master
 
+  # Separate workflow just for version tag releases.
+  tagged-release:
+    jobs:
+      - test:
+          filters: &version-tag-only
```

Comment:
nicely done!

Reply:
:smile:  anchors are so helpful in reducing noise